sdp/processors/toloka/create_project.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+
+import toloka.client
+import toloka.client.project.template_builder
+
+from sdp.logging import logger
+from sdp.processors.base_processor import BaseProcessor
+
+
+class CreateTolokaProject(BaseProcessor):
+    def __init__(
+        self,
+        project_name: str,
+        project_description: str,
+        project_instructions: str,
+        API_KEY: str = None,
+        platform: str = None,
+        save_api_key_to_config: bool = False,  # Parameter to control saving API key
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        self.API_KEY = API_KEY or os.getenv('TOLOKA_API_KEY')
```

Comment:
Can we just use os.getev for all these processors? And not read from the config?